nedele.html

```diff
@@ -0,0 +1,179 @@
+              />
+            </g>
+          </svg>
+          <h1>Start@FIT</h1>
+        </a>
+      </div>
+
+      <button id="menuButton" onclick="toggleNav()" aria-label="Toggle menu">
+        <i class="fa-solid fa-bars"></i>
+      </button>
+    </nav>
+
+    <div id="menu" class="hidden top-margin">
+      <div class="menu-content">
+        <ul class="centering plain">
+          <a href="/#Program" onclick="toggleNav()"
+            ><li>Přehled programu</li></a
+          >
+          <hr />
+          <a href="ctvrtek"><li>Čtvrtek</li></a>
+          <a href="patek"><li>Pátek</li></a>
+          <a href="Neděle"><li>Sobota</li></a>
+          <a href="nedele"><li>Neděle</li></a>
+          <hr />
+          <a href="photos"><li>Fotografie z&nbsp;minulých let</li></a>
+        </ul>
+      </div>
+    </div>
+
+    <article>
+      <div class="section top-margin highlight">
+        <h2 class="day-title">
+          <a href="sobota"><i class="fa-solid fa-arrow-left"></i></a>
+          <span> Neděle </span>
+          <a href=""><i class="fa-solid fa-arrow-right invisible"></i></a>
+        </h2>
+
+        <p class="centering">15. září/septembra 2023</p>
+
+        <div class="inner">
+          <p>
+            Nedělní ráno začneme <b>Snídaní se Studentskou unií</b> v Kachničce.
+          </p>
+          <p>
+            Pak volně navážeme hrou <b>Brnění</b>, kde budete mít příležitost
+            prozkoumat to nejlepší město pro studium – Brno.
+          </p>
+          <p>
+            A večer se sejdeme v <b>Božetěchově sadu</b>, kde si můžeme společně
+            posedět a něco si ugrilovat.
+          </p>
+        </div>
+      </div>
+
+      <div class="section">
+        <div class="inner">
+          <div class="centering">
+            <p><b>Konkrétní informace ještě upřesníme.</b></p>
+          </div>
+        </div>
+      </div>
+
```

Comment:
Stejná poznámka jako na sobotní stránce, co tu přidat fotku/něco ať to nepůsobí tak prázdně?